tests/unit/adapters/test_user_options.py

```diff
@@ -119,3 +119,72 @@ def test_respects_passed_filename(passed_filename, expected_foo_value):
     else:
         options = reader.read_options(config_filename=passed_filename)
         assert expected_options == options
+
+
+@pytest.mark.parametrize(
```

Comment:
We should also have some functional tests to check everything's plumbed in correctly, see [here](https://github.com/kasium/import-linter/blob/b59a76d79bc8c088158f49b24b2d53f1eeb58535/tests/functional/test_lint_imports.py#L19).

The best approach I can think of is to have a test that runs on a TOML file both with toml installed and not installed, so we can verify what the behaviour is.

Reply:
Please refer to my comment at the tox file